docs/rebuild_admin.rst

```diff
@@ -379,54 +368,65 @@ previous steps. To do so, connect to the Tor network on the
 .. note:: The ``./securedrop-admin setup`` command may take several minutes to complete, and may
  fail due to network issues. If it fails, it's safe to run again.
 
-When prompted, fill in the values retrieved in the previous steps.
+The ``sdconfig`` command will prompt you to fill in configuration details
+about your instance. Use the information retrieved in the previous steps.
+When prompted whether or not to enable SSH-over-Tor, type **no** to ensure
+that SSH-over-LAN remains enabled.
 
-Once complete, configure the *Admin Workstation* using the command:
+Next, back up the Application server by running the following command in the terminal:
 
 .. code:: sh
 
-  ./securedrop-admin tailsconfig
+ ./securedrop-admin backup
 
-This will set up desktop shortcuts for the *Source* and *Journalist Interfaces*,
-and configure adminstrative access to the servers.
+Ensure the backup command completes successfully.
 
-Once the ``./securedrop-admin tailsconfig`` command is complete:
- - verify that the ``Hostname`` references in ``~/.ssh/config`` have been updated
-   to refer to Onion URLs instead of direct IP addresses,
- - verify that you can connect to
-   the servers using ``ssh app`` and ``ssh mon``, accepting the host verification
-   prompt if necessary,
- - and verify that the desktop shortcuts for the *Source* and *Journalist Interfaces*
-   work correctly, opening their respective homepages in Tor Browser.
+Step 6 (Recommended): Re-enable SSH-over-Tor
+============================================
 
-Next, back up the servers by running the following command in the terminal:
+We recommend enabling SSH over Tor and disabling SSH-over-LAN.
+To do so, run:
 
 .. code:: sh
 
- ./securedrop-admin backup
+ ./securedrop-admin sdconfig
 
+Press "Enter" to accept the prepopulated values until you reach the
+SSH-over-Tor settings, this time typing **yes** to enable SSH-over-Tor.
 
-Step 6: Run the ``./securedrop-admin install`` command
-======================================================
+Step 7: Use the installer to complete the configuration
```

Comment:
I changed this section title from "fetch the configuration" in 90668bbda80ecd95f4f222d31767c01c6b7b2242 because the installation playbook may also change the Tor onion service authentication details present on the server, making it more than a fetch operation.